### PR TITLE
AA-381: Fix shorten welcome message bug

### PR DESCRIPTION
**Pull request by ciduarte**
- Switched from 200 character count to 100 word limit for parity with old experience

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,8 @@
     "react-share": "4.2.1",
     "redux": "4.0.5",
     "regenerator-runtime": "0.13.7",
-    "reselect": "4.0.0"
+    "reselect": "4.0.0",
+    "truncate-html": "^1.0.3"
```
Comment (mikix) [inline package.json:62]: A. I hate how json doesn't use trailing commas.
B. We don't generally use the ^ syntax with renovate (and renovate will drop it next time it look at these I think)

**Review by mikix — APPROVED** [review]
Seems simple enough! Good find!

**Review by Dillon-Dumesnil — APPROVED** [review]
It all looks good to me.

Should we add in a test to cover the case that brought up this issue in the first place? Like ensure all tags are appropriately closed as part of truncating? Or that we are accurately ignoring certain tags? Not sure if that's just getting into the part of testing that a library does what it should. Just thinking out loud.

File: package.json (hunk 1 above)
Comment (ciduarte) [reply package.json:62]: Ooh is it worth manually dropping the ^ to be safe?

Comment (mikix) [reply package.json:62]: Naw I think it's fine to leave in. Just talking out loud over here

**Comment by ciduarte** [comment]
> It all looks good to me.
> 
> Should we add in a test to cover the case that brought up this issue in the first place? Like ensure all tags are appropriately closed as part of truncating? Or that we are accurately ignoring certain tags? Not sure if that's just getting into the part of testing that a library does what it should. Just thinking out loud.

Nothing actually triggered us noticing this bug, I just realized that it was being truncated incorrectly when writing the test for this. I think that might fall into testing the library too much, and there are some complications with testing the welcome message html itself because it all gets put into an iframe.

**Pull request merged**